src/test/kotlin/io/snyk/plugin/ui/toolwindow/LabelProviderTest.kt

```diff
@@ -93,6 +93,17 @@ class LabelProviderTest {
         verifyLinkLabelCreated(url)
     }
 
+    @Test
+    fun `getVulnerability should provide an uppercase label and keep vulnerability ID case sensitivity in url`() {
+        val id = "vuln123"
+
+        val output: JLabel = LabelProvider().getVulnerabilityLabel(id)
```

Comment:
Minor: you only need to specify type if no assignment made.

Reply:
oh, I made it consistent with the cases in this class. I'll keep it as-is, if you don't mind 👼 Too much git commit/rebase work for me 😅

Reply:
actually changed it, since required to do a change as per your another comment :)